translations/locales/fr-CA/translations.json

```diff
@@ -0,0 +1,591 @@
+{
+    "Nav": {
+      "File": {
+        "Title": "Fichier",
+        "New": "Nouveau",
+        "Share": "Partager",
+        "Duplicate": "Dupliquer",
+        "Open": "Ouvrir",
+        "Download": "Télécharger",
+        "AddToCollection": "Ajouter à la collection",
+        "Examples": "Exemples"
+      },
+      "Edit": {
+        "Title": "Edition",
+        "TidyCode": "Nettoyer le code",
+        "Find": "Rechercher",
+        "FindNext": "Prochaine correspondance",
+        "FindPrevious": "Correspondance précédente"
+      },
+      "Sketch": {
+        "Title": "Croquis",
+        "AddFile": "Nouveau fichier",
+        "AddFolder": "Nouveau dossier",
+        "Run": "Exécuter",
+        "Stop": "Arrêter"
+      },
+      "Help": {
+        "Title": "Aide",
+        "KeyboardShortcuts": "Raccourcis clavier",
+        "Reference": "Référence",
+        "About": "À propos"
+      },
+      "Lang": "Langue",
+      "BackEditor": "Retour à l'éditeur",
+      "WarningUnsavedChanges": "Êtes-vous certain de vouloir quitter cette page? Vous avez des changements non enregistrés.",
+      "Login": "Se connecter",
+      "LoginOr": "ou",
+      "SignUp": "S'inscrire",
+      "Auth": {
+        "Welcome": "Bienvenue",
+        "Hello": "Bonjour",
+        "MyAccount": "compte",
+        "My": "Mon",
+        "MySketches": "Mes croquis",
+        "MyCollections": "Mes collections",
+        "Asset": "Ressources",
+        "MyAssets": "Mes ressources",
+        "LogOut": "Se déconnecter"
+      }
+    },
```

Comment:
If you look at the Khttps://github.com/processing/p5.js-web-editor/blob/develop/translations/locales/en-US/translations.json#L50, there's a section here called "CodeMirrorFindAndReplace":
```json
"CodemirrorFindAndReplace": {
    "ToggleReplace": "Toggle Replace",
    "Find": "Find",
    "FindPlaceholder": "Find in files",
    "Replace": "Replace",
    "ReplaceAll": "Replace All",
    "ReplacePlaceholder": "Text to replace",
    "Regex": "Regular expression",
    "CaseSensitive": "Case sensitive",
    "WholeWords": "Whole words",
    "Previous": "Previous",
    "Next": "Next",
    "NoResults": "No results",
    "Close": "Close"
  }
```